app/tests/test_chat_sessions_integration.py

```
@@ -0,0 +1,335 @@
+                    "/api/v1/chat-sessions/1",
+                    json={
+                        "title": "Updated Session Title",
+                        "metadata": {"updated": True}
+                    },
+                    headers={"Authorization": "Bearer test_token"}
+                )
+                
+                assert response.status_code == 200
+                data = response.json()
+                assert data["title"] == "Updated Session Title"
+                assert data["metadata"]["updated"] == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if data["metadata"]["updated"]:` for truth checks

Replace with `data["metadata"]["updated"]`

(E712)